packages/sources/view-function-multi-chain/test/integration/adapter.test.ts

```diff
@@ -22,6 +25,16 @@ describe('execute', () => {
     process.env.ETHEREUM_GOERLI_RPC_URL =
       process.env.ETHEREUM_GOERLI_RPC_URL ?? 'http://localhost:8554'
     process.env.ETHEREUM_GOERLI_CHAIN_ID = process.env.ETHEREUM_GOERLI_CHAIN_ID ?? '5'
+
+    process.env.SARKNET_MAINNET_RPC_URL =
```

Comment:
I'm not sure how this test is currently working as the `env` var wouldn't match the `networkName` 🤔

```suggestion
    process.env.STARKNET_MAINNET_RPC_URL =
```

Reply:
Ah yes, will fix that typo. I think it worked for me as I had the STARKNET_MAINNET_RPC_URL env variable defined already.